app/forms/sufia/forms/batch_edit_form.rb

```diff
@@ -1,9 +1,46 @@
 module Sufia
   module Forms
-    class BatchEditForm < CurationConcerns::Forms::FileSetEditForm
-      self.terms = [:resource_type, :title, :creator, :contributor, :description,
-                    :keyword, :rights, :publisher, :date_created, :subject, :language,
+    class BatchEditForm < CurationConcerns::Forms::WorkForm
+      self.terms = [:creator, :contributor, :description,
+                    :keyword, :rights, :publisher, :subject, :language,
                     :identifier, :based_near, :related_url]
+      self.required_fields = []
+
+      self.model_class = ::GenericWork
+
+      attr_accessor :names
+
+      def initialize(model, current_ability, batch)
+        super(model, current_ability)
+
+        # need @ so that we are initializing the class instead of a local
```

Comment:
I suspect you can drop this comment, since setting instance variables in `#initialize` is a very common Ruby pattern.